core-java-modules/core-java-concurrency-advanced/src/main/java/com/baeldung/concurrent/atomicvsvolatile/SafeAtomicCounter.java

```diff
@@ -0,0 +1,16 @@
+package com.baeldung.concurrent.atomicvsvolatile;
```

Comment:
maybe for better readibility:
```suggestion
package com.baeldung.concurrent.atomic_vs_volatile;
```